bin/alphanet/Cargo.toml

```diff
@@ -14,19 +14,20 @@ workspace = true
 [dependencies]
 alphanet-node.workspace = true
 tracing.workspace = true
-reth.workspace = true
+reth.workspace = true                                             # not needed
 reth-cli-util.workspace = true
 reth-node-optimism.workspace = true
+reth-optimism-cli = { workspace = true, features = ["optimism"] }
 reth-optimism-rpc.workspace = true
 clap = { workspace = true, features = ["derive"] }
 
 [target.'cfg(not(windows))'.dependencies]
 tikv-jemallocator = { version = "0.5", optional = true }
 
 [features]
-default = ["jemalloc", "reth/optimism", "reth-node-optimism/optimism"]
+default = ["jemalloc", "reth-node-optimism/optimism"]
 
-asm-keccak = ["reth/asm-keccak"]
+asm-keccak = ["reth/asm-keccak"] # how do we enable this now
```

Comment:
wdym how do we enable this now?

Reply:
we don't use `reth` as a dep anymore so after i remove it i'm not sure where to enable `asm-keccak`

Reply:
needs node-core for observability and alloy-primitives

Reply:
ah, we do actually need to enable it in `reth-node-optimism`